Foundation/NSObjCRuntime.swift

```diff
@@ -11,11 +11,139 @@
 import CoreFoundation
 
 #if os(OSX) || os(iOS)
+import Darwin
+#elseif os(Linux)
+import Glibc
+#endif
+
+#if os(OSX) || os(iOS)
 internal let kCFCompareLessThan = CFComparisonResult.CompareLessThan
 internal let kCFCompareEqualTo = CFComparisonResult.CompareEqualTo
 internal let kCFCompareGreaterThan = CFComparisonResult.CompareGreaterThan
 #endif
 
+internal enum NSObjCType : UnicodeScalar {
```

Comment:
These are simple objc types because you can have for example `"^@"` or `"{dddd}"` etc, and probably we should prefix this with an underscore to drive home the point that this is internal